dali/test/python/test_dali_tf_dataset_shape.py

```diff
@@ -166,7 +172,9 @@ def define_graph(self):
 
 def test_multiple_input_valid_shapes():
     for batch in [1, 10]:
-        for shapes in [None, (None, None), ((batch, 200, 200, 3), None), (None, (batch, 1)), (None, (batch,))]:
+        for shapes in [
+            None, (None, None), ((batch, 200, 200, 3), None), (None, (batch, 1)), (None, (batch,))
```

Comment:
Actually, given the number of commas and braces:
```suggestion
            None,
            (None, None),
            ((batch, 200, 200, 3), None),
            (None, (batch, 1)),
            (None, (batch,))
```